pronto-jscs.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_runtime_dependency('pronto', '~> 0.5.0')
+  spec.add_runtime_dependency('pronto', '~> 0.5')
```

Comment:
Ah, wasn't aware of that. Anyway, #1 prevents using pronto-jscs with pronto 0.6.x for now.